doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -13,7 +13,7 @@ sections explain how to re-build in the preferred configuration and how to
 enable more features, such as distributed memory execution.
 
 
-0) See `doc/prereqs.rst`_ for more information about system tools and
+0) See :ref:`prereqs.rst <readme-prereqs>` for more information about system tools and
```

Comment:
This doesn't look like it renders properly when viewed online (https://github.com/ben-albrecht/chapel/blob/4ba85c0d15d2756b2eb95c80e68855e3b3d5acd7/doc/rst/usingchapel/QUICKSTART.rst) -- is that to be expected?